src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-logs.mdx

```diff
@@ -1,47 +1,176 @@
-
-The [Logs in Context with Log4j2 example in GitHub](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/other-examples/java/logs-in-context-log4j2) demonstrates an end-to-end working example for a simple Java application using Log4j2.
-
-### View OpenTelemetry logs [#view-logs]
-
-Here are two ways you can view logs:
-
-* Look in our [logs UI](/docs/logs/log-management/ui-data/use-logs-ui/).
-* If your logs are correlated with an application, view them in the [context of the application](/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic#logs).
-
-### The time field [#log-time]
-
-The `timeUnixNano` field is optional according to the OpenTelemetry specification for log data. When `timeUnixNano` is not present New Relic will use the time that the data was received for the New Relic log timestamp.
+New Relic supports two primary workflows for ingesting logs generated with OpenTelemetry:
+
+* [Direct to collector](https://opentelemetry.io/docs/specs/otel/logs/#direct-to-collector):
+    * Applications can send logs directly to the dedicated New Relic OpenTelemetry Log Protocol (OTLP) endpoint. This streamlined approach minimizes additional components.
+    * Refer to the relevant [OpenTelemetry language documentation](https://opentelemetry.io/docs/languages/) for specific implementation details.
+* [Via a file or `stdout`](https://opentelemetry.io/docs/specs/otel/logs/#via-file-or-stdout-logs):
+    * This method involves scraping application logs written to files or standard output (`stdout`).
+    * The OpenTelemetry collector is typically used for this task. Scraped logs are then forwarded to the New Relic OTLP endpoint.
+    * In-depth configuration information can be found in the following OpenTelemetry resources:
+        * [OpenTelemetry collector](https://opentelemetry.io/docs/collector/)
+        * [Filelog receiver with kubernetes](https://opentelemetry.io/docs/kubernetes/collector/components/#filelog-receiver)
+        * [A practical example demonstrating this workflow for Java applications using Log4j2](https://github.com/newrelic/newrelic-opentelemetry-examples/tree/main/other-examples/java/logs-in-context-log4j2)
+
+This documentation focuses on how New Relic processes OpenTelemetry logs received through its dedicated OTLP endpoint. Regardless of the chosen collection method, successful integration requires configuring your log source to export logs to this endpoint. Make sure to review the [endpoint configuration requirements](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-otlp/#configure-endpoint-port-protocol) before proceeding.
+
+For information on configuring services with OpenTelemetry logs, see [OpenTelemetry APM monitoring](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/get-started/apm-monitoring/opentelemetry-apm-intro/).
```

Comment:
@jack-berg I think we need more info/preface for this link. Why would they want more info on configuring services with OTEL logs? Do they need to read that doc before returning to this one? More context would be helpful here